### DISCUSSION TITLE
Macro help

### DISCUSSION THREAD

**Discussion by jk8104** [chhoumann/quickadd#315] [Q&A]
I'm trying to create some of my first macros. (non-coder) Use case=I want to create series of quickadd actions that captures a note with the time stamp to my # NOTES section of my daily notes AND at the same time runs a Template action that creates a new meeting notes note.

In essence, I want the outcome in my daily note to be as follows:

# NOTES
- **{{DATE:hh:mm A}}** - [[Link to Meeting Note Here]]

Please advise! Thanks in advance!

**Comment by chhoumann** [comment]
Hi @jk8104 

That's one of my most-used workflows, actually!
In this case, it doesn't matter much in which order the steps of the macro runs.

You can mostly set up the template however you like, but the file name has to have a specific format. E.g. you can use ``{{VALUE:Meeting name}}`` or similar. Just keep it accessible, so we can use it later exactly.

Then, in the capture, you can use insert after set to `# Notes` and capture format set to: `- {{DATE:hh:mm A}} - [[{{VALUE:Meeting name}}]]`.
Now it'll capture a link to the meeting note.
